kloppy/tests/test_wyscout.py

```diff
@@ -86,7 +86,7 @@ def test_correct_v2_deserialization(self, event_v2_data: Path):
             == DuelType.SLIDING_TACKLE
         )
         assert (
```

Comment:
Would it be possible to add a test for a `FoulCommittedEvent` followed by a `CardEvent`?